vendor/bat-native-ledger/src/bat/ledger/internal/server/api/endpoints/get_parameters.cc

```diff
@@ -139,4 +170,31 @@ ledger::Result ParseParameters(
   return ledger::Result::LEDGER_OK;
 }
 
-}  // namespace braveledger_response_util
+void GetParameters::CreateRequest(GetParametersCallback callback) {
+  auto url_callback = std::bind(&GetParameters::OnRequest,
+      this,
+      _1,
+      callback);
+  ledger_->LoadURL(GetUrl(), {}, "", "", ledger::UrlMethod::GET, url_callback);
```

Comment:
Not specific to this PR, but it seems like we always have a lot of parameters in `LoadURL` that don't usually vary from the norm. If we rearranged those to be at the end of the function, we could make things a little less noisy in the common case.

Reply:
agree, @emerick can you please log an issue for it

Reply:
Created https://github.com/brave/brave-browser/issues/11172.